fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/derby/DerbyResourceDAO.java

```diff
@@ -504,4 +568,18 @@ public int getOrCreateResourceType(String resourceTypeName, Connection conn) thr
 
         return result;
     }
+
+    /**
+     * Safely close the ResultSet
+     * @param rs
+     */
+    private void safeClose(ResultSet rs) {
+        try {
+            if (rs != null) {
+                rs.close();
+            }
+        } catch (SQLException x) {
+            // NOP
+        }
+    }
```

Comment:
I thought ResultSets autoclose out of scope?

Reply:
actually, where is this used?

Reply:
Yeah, it's not. I added this as part of debugging to see if actively closing the ResultSet would make a difference (thinking there might be a bug lurking underneath even though closing the statement should close any associated ResultSet). I'll remove.